rfhub2/model/__init__.py

```diff
@@ -86,5 +86,21 @@ class Config:
         orm_mode = True
 
 
+class KeywordStatisticsList(BaseModel):
+    __root__: List[KeywordStatistics]
+
+    @staticmethod
+    def of(items: List[KeywordStatistics]) -> "KeywordStatisticsList":
+        return KeywordStatisticsList(__root__=items)
+
+    @staticmethod
+    def one(item: KeywordStatistics) -> "KeywordStatisticsList":
+        return KeywordStatisticsList(__root__=[item])
+
+
```

Comment:
What do those methods do?

Reply:
I wanted to add such helpers so we do not need to explicitly instantiate this custom root type everywhere: https://pydantic-docs.helpmanual.io/usage/models/#custom-root-types